src/components/TabMenu/index.js

```diff
@@ -26,11 +26,7 @@ TabMenu.defaultProps = {
   className: '',
 };
 
-/*
- * SimpleComponent naming convention is deprecated due to confusion.
- * Use EasyComponent moving forward.
- */
-export const SimpleTabMenu = ({ tabs, activeTab, size = 'sm', ...props }) => {
+export const EasyTabMenu = ({ tabs, activeTab, size = 'sm', ...props }) => {
```

Comment:
I changed the primary exported component to be `EasyTabMenu` so it'll be easier to rip out the old one.

Sorry for stepping on your toes, @chelshaw! I wrote most of this last night on the `Pill` branch when adding `EasyPill`, for consistency. Just renamed stuff.

Reply:
I thought... i already did this in the previously merged code? but good call on switching the default